README.md

```diff
@@ -1,41 +1,143 @@
 # Eno
 
-## What is Eno?
+Compose Kubernetes deployments.
 
-Eno is a new take on configuration management for Kubernetes.
+- 🎹 **Synthesize**: generate manifests dynamically in short-lived pods
+- ♻️ **Reconcile**: apply the generated configurations to one (or more!) clusters
+- 🏃‍➡️ **React**: track and aggregate the readiness state of managed resources
+ 
 
-- Generate configurations using short-lived pods, language-agnostic
-- Dynamically regenerate when input resources change (without writing a controller!)
-- Manage risk when updating many similar configurations
-- Define complex ordering relationships
-- Control any number of low-trust clusters from a single management cluster
-- Support high object cardinality (10s of thousands)
+## What can Eno do?
+
+- Magically regenerate configurations when their inputs change
+- Safely roll out changes that impact many instances of a configuration
+- Support deployments larger than apiserver's 1.5MB resource limit
+- Define complex ordering relationships between resources
+- Patch resources without taking full ownership
+
+## Docs
+
+- [Synthesizers](./docs/synthesizer-api.md)
+- [Inputs](./docs/inputs.md)
+- [Ordering](./docs/ordering.md)
+- [Symphonies](./docs/symphony.md)
+- [Generated API Docs](./docs/api.md)
 
 ## Getting Started
 
+### 1. Install Eno
+
 ```bash
 export TAG=$(curl https://api.github.com/repos/Azure/eno/releases | jq -r '.[0].name')
 kubectl apply -f "https://github.com/Azure/eno/releases/download/${TAG}/manifest.yaml"
 ```
 
-Next, create a minimum viable Eno configuration to make sure everything works.
-This manifest will create a configmap called "some-config" in the default namespace.
+### 2. Create a Synthesizer
+
+Synthesizers reference a container image that implements a [KRM function](https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/functions-spec.md).
+This example uses a small bash script, but you will probably want to use `github.com/Azure/eno/pkg/function`.
 
 ```bash
-kubectl apply -f "https://raw.githubusercontent.com/Azure/eno/refs/heads/main/examples/minimal/example.yaml"
+kubectl apply -f - <<YAML
+apiVersion: eno.azure.io/v1
+kind: Synthesizer
+metadata:
+  name: getting-started
+  namespace: default
+spec:
+  image: docker.io/ubuntu:latest
+  refs:
+    - key: config
+      resource:
+        group: "" # core
+        version: v1
+        kind: ConfigMap
+  command:
+  - /bin/bash
+  - -c
+  - |
+    replica_count=\$(sed -n 's/.*"replicas":"\([^"]*\)".*/\1/p')
```

Comment:
Worth adding a comment to demistify this? At least mention that this is equivalent to using `function` to read an input?